include/glow/Support/Support.h

```diff
@@ -110,6 +110,10 @@ class DescriptionBuilder {
 void report(const char *msg);
 inline void report(const std::string &str) { report(str.c_str()); }
 inline void report(llvm::StringRef str) { report(str.data()); }
+
+/// Printf-like formatting for std::string.
+const std::string strFormat(const char *format, ...)
+    __attribute__((__format__(__printf__, 1, 2)));
```

Comment:
what does this __attribute__ do?

Reply:
It tells GCC/Clang to verify that passed varargs conform to the format string.